ifrs17/Import/Importers.ipynb

```diff
@@ -1334,13 +1372,26 @@
                 "\n    if(Activity.HasErrors()) return Activity.Finish();",
                 "\n    ",
                 "\n    var hasAccidentYearColumn = dataSet.Tables[ImportFormats.Cashflow].Columns.Any(x => x.ColumnName == nameof(RawVariable.AccidentYear));",
+                "\n    var hasPeriodicityColumn = dataSet.Tables[ImportFormats.Cashflow].Columns.Any(x => x.ColumnName == nameof(CashFlowPeriodicity));",
+                "\n    var hasInterpolationMethod = dataSet.Tables[ImportFormats.Cashflow].Columns.Any(x => x.ColumnName == nameof(InterpolationMethod));",
                 "\n    ",
                 "\n    var importLog = await Import.FromDataSet(dataSet)",
                 "\n        .WithType<RawVariable> ( (dataset, datarow) => {",
                 "\n            var aocType = datarow.Field<string>(nameof(RawVariable.AocType));",
                 "\n            var novelty = datarow.Field<string>(nameof(RawVariable.Novelty));",
                 "\n            var dataNode = datarow.Field<string>(nameof(DataNode));",
                 "\n            ",
+                "\n            // CashflowPeriodicity given by the Cashflows or else taken from the SingleDataNodeParameters",
+                "\n            var cashFlowPeriodicity = hasPeriodicityColumn && (Enum.TryParse(datarow.Field<string>(nameof(CashFlowPeriodicity)), out CashFlowPeriodicity cfp))",
```

Comment:
Parsing of these columns should be similar to the one in SingleDataNodeParameter

Reply:
open a new issue to cover the testing of these new attributes in cashflows: #311